### ISSUE TITLE
How to get the exact version running ?

### ISSUE THREAD

**Issue by autourdupc** [patience4711/RPI-APS-inverters#131]
Hi !

How can I get the exact version running ?
I mean that I know that the version is "firmware version = RPI-ECU-v4_2", but is it 4.2a, 4.2b or ... ?

I think that the exact version should be displayed in the "info.php" page.

Regards,
Laurent

**Comment by patience4711** [comment]
install update 4_3.

**Comment by autourdupc** [comment]
Hello @patience4711 

Just updated with V4.3.
Still remains on 4.2 !

![image](https://github.com/user-attachments/assets/951c38f6-ae3e-4841-9895-0404eb2775bc)

![image](https://github.com/user-attachments/assets/8e9953d2-c8ad-4ceb-8921-2563b91471dd)

And just after a reboot; it is not better.
Note also a little confusion on texts 
![image](https://github.com/user-attachments/assets/b92d43a9-b606-4c84-9b47-c591cd7cca18)

**Comment by patience4711** [comment]
I checked and all seems oke to me. 
During update a script writes "echo 'RPI-ECU-v4_3' > /var/www/static_data/swVersion.txt"

Can you check that this file is oke? If not, the permissions on that file are probably not r/w.

Or your infopage doesn't read that file. Or you get a cached version of the infopage.

**Comment by patience4711** [comment]
Yes i get the point,  the script doesn't write to the file. You didn't aswer my questions.

You can **echo 'RPI-ECU-v4_3' > /var/www/static_data/swVersion.txt**  yourself and see if that works.

**Comment by autourdupc** [comment]
It works using the echo cmde...

![image](https://github.com/user-attachments/assets/67809eac-1b1b-4d46-b556-5d69c1dd4ce5)

**Comment by patience4711** [comment]
I just installed image 4_2 on an sd and updated to 4_3. No problems found, the version shown in the infopage = 4_3

**Comment by autourdupc** [comment]
Hum... I did not change anything on the 4_2... Just added my personal power script, nothing else !
Very strange

**Comment by patience4711** [comment]
Your echo command worked, and that is exactly the command the script runs so we have another inexplicable phenomenon.
In my case the update log showed that the command was issued. I repeated with the file i downloaded on github to be sure. 
So the only possibility is that the browsercache showed the old page.

![updatelog_1](https://github.com/user-attachments/assets/916988ec-e1c0-40da-9589-5d1af71d1a98)
